travvy/src/components/Landing.vue

```diff
@@ -4,8 +4,16 @@
     <img src="../images/map.jpg" alt="Picture of Map"/>
     <br>
     <br>
+    <!-- Button to go to the log inpage --> 
+    <router-link to="/Home" tag="button" class="button"><span>Log In</span></router-link>
+    <br>
+    <br>
+    <!-- Button to go to the sign up page --> 
+    <router-link to="/Home" tag="button" class="button"><span>Sign Up</span></router-link>
```

Comment:
The same as above - it looks good for now. Will need to change the router link to the sign up page once this page is created.